src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,29 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from '../../app/hooks';
+import { actions } from '../../features/filter';
 
 export const TodoFilter: React.FC = () => {
+  const dispatch = useDispatch();
+  const query = useAppSelector(state => state.filter.query);
+  const status = useAppSelector(state => state.filter.status);
```

Comment:
Can't you do smth like this? 
```suggestion
  const { query, status } = useAppSelector(state => state.filter);
```

Reply:
done